ceph/ceph_admin.py

```diff
@@ -224,6 +224,73 @@ def manage_hosts(self, **kwargs):
             # todo: add code for removing host
             raise NotImplementedError
 
+    def label_host(self):
+        node_mon = self.cluster.get_nodes(role="mon")
+        node_mgr = self.cluster.get_nodes(role="mgr")
+        node_osd = self.cluster.get_nodes(role="osd")
+        node_mds = self.cluster.get_nodes(role="mds")
+        node_rgw = self.cluster.get_nodes(role="rgw")
+        node_client = self.cluster.get_nodes(role="client")
+        # self.ceph_iscsi(self.cluster.get_nodes(role="iscsi"))
+        for node in node_mon:
```

Comment:
IMO, we could have a common/private method that takes the node & label.. for example
```
def _apply_label(self, node_name, label):
  self.shell(remote=self.installer, args= [ 'ceph', 'orch', 'host', 'label' , 'add', node_name, label])
```